dev.sh

```diff
@@ -7,6 +7,11 @@ echo "Creating dev environment in ./venv..."
 python3.7 -m venv venv
 . venv/bin/activate
 pip3.7 install -U pip setuptools
+
+# https://github.com/qtile/qtile/issues/994
```

Comment:
I'd rather link to the explanations directly: https://github.com/qtile/qtile/issues/994#issuecomment-497984551